examples/vm_scheduling/reinforcement_learning/ac/agent/models/combine_net.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
+import torch.nn as nn
+
+from maro.rl import FullyConnectedBlock
+
+
+class CombineNet(FullyConnectedBlock):
+    """Fully connected network with optional batch normalization, activation and dropout components.
+
+    Args:
+        name (str): Network name.
+        input_dim (int): Network input dimension.
+        output_dim (int): Network output dimension.
+        hidden_dims ([int]): Dimensions of hidden layers. Its length is the number of hidden layers.
+        activation: A ``torch.nn`` activation type. If None, there will be no activation. Defaults to LeakyReLU.
```

Comment:
it is not `torch.nn` type here, it is the type pre-defined in the rl toolkit. (You need to find out where is the definition by yourself)